script/Ciel/Day 1/01_01_CIEL01_2_3.txt

```diff
@@ -1226,7 +1226,7 @@ I mean, did you not get along with your previous family...?"
 [sha:f3a31ca7d9a1cd58f43521608b71f11632be82d1]{
 -- Page 158, Offset 862.
 -- 「いや、そんなコトはないっスよ。こいつ、有間の親御さんとは何の問題もないもんな。みんな超いい人たちだし。いや、有間の親父さんはわりと怖いけど。
-"No, that's not true. This guy has no problems with the Arima family. They're all super nice people. Well, old man Arima is rather scary, though.
+"No, that's not true. This guy has no problems with the Arima family. They're all super nice people. Well, papa Arima is rather scary, though.
```

Comment:
Maybe prefer original, old man Surname feels like how you'd refer to the scary old neighbour that doesn't return the football when it goes over the fence. Papa also just reads very infantile to me

Reply:
indeed. papa isn't something people say like this

Reply:
The thing was mostly that 'old man Arima' sounded to me like it was the grandpa of the family or something when its quite clearly just the dad of the family.

Reply:
Maybe encode with prefix?
> Well, I guess Mr. Arima himself is kinda scary, though.